x-pack/plugins/security_solution/docs/prebuilt_rules_customization_rfc.md

```diff
@@ -0,0 +1,2867 @@
+      <ul><li><b>1st elem: (A B C - keep current) </li><li>2nd elem: (A _ A - remove element)</b></li></ul>
+      And:
+      <ul><li><b>1st elem: (A B C - keep current) </li><li>2nd elem: (A _ B - remove element</b></li></ul>
+      </td>
+      <td>
+      <pre>
+[
+  logs-*,
+  logstash-*
+]
+        </pre>
+      </td>
+      <td>
+        <pre>
+[
+  cluster_one:logs-*,
+]
+        </pre>
+      </td>
+      <td style="border-right:3px solid black">
+      <pre>
+[
+  logs-new-*,
+  logstash-*
+]
+        </pre>
+        </td>
+      <td>
+      <pre>
+[
+  cluster_one:logs-*,
+]
+        </pre>
+      </td>
+    </tr>
+    <tr>
+      <td>
+      <pre>
+[
+  {
+    ecs: true,
+    name: "event.action",
+    type: "unknown"
+  },
+  {
+    ecs: true,
+    name: "process.executable",
+    type: "string"
+  }
+]
+        </pre>
+      </td>
+      <td>
+        <pre>
+[
+  {
+    ecs: true,
+    name: "event.UPDATED",
+    type: "unknown"
+  }
+]
+        </pre>
+      </td>
+      <td style="border-right:3px solid black">
+      <pre>
+[
+  {
+    ecs: true,
+    name: "event.action",
+    type: "keyword"
+  },
+  {
+    ecs: true,
+    name: "process.executable.ip",
+    type: "ip"
+  }
+]
+        </pre>
+        </td>
+      <td>
+      <pre>
+[
+  {
+    ecs: true,
+    name: "event.UPDATED",
+    type: "unknown"
+  }
+]
+        </pre>
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+
+##### MITRE ATT&CK framework (`threat` field)
+
+The `threat` field has a specific [schema](https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/common/api/detection_engine/model/rule_schema/common_attributes.gen.ts#L250) that can be handled with its own algorithm in case of conflicts, in order to try to obtain the most reasonable merge between versions. 
+
+**TODO: write algorithm specific to this field**
+
+  <table>
+    <thead>
+      <tr>
+        <th style="border-right:3px solid black">Use case</th>
+        <th>Base version</th>
+        <th>Current version</th>
+        <th style="border-right:3px solid black">Target version</th>
+        <th>Merged version (output)</th>
+      </tr>
+    </thead>
+    <tbody valign="top">
+    <tr>
+      <td style="border-right:3px solid black">Keep user customizations on conflicts, but update sections with no conflicts</td>
+      <td>
+      <pre>
+[{
+    "framework":"MITRE ATT&CK",
+    "tactic":{
+      "id":"TA0003",
+      "name":"Persistence",
+      "reference":"https://attack.mitre.org/tactics/TA0003/"
+    },
+    "technique":[
+      {
+          "id":"T1098",
+          "name":"Account Manipulation",
+          "reference":"https://attack.mitre.org/techniques/T1098/"
+      }
+    ]
+}]
+        </pre>
+      </td>
+      <td>
+        <pre>
+[{
+    "framework":"MITRE ATT&CK",
+    "tactic":{
+      "id":"TA0003",
+      "name":"My customized name",
```

Comment:
Example here doesn't look correct. Users can only add and remove MITRE ATT&CK tacticts, techniques and subtechniques from the a specific MITRE ATT&CK version Kibana is aware of. `name` and `reference` fields are here only to simplify name retrival.

Reply:
Removing whole Mitre section for now.